frontend/cypress/tests/codeView/codeView_codeHighlighting.cy.js

```diff
@@ -0,0 +1,19 @@
+describe('code highlighting works properly', () => {
+  it('line numbers should all have the same colour', () => {
+    cy.get('.icon-button.fa-code')
+      .should('exist')
+      .first()
+      .click();
+
+    cy.get('#tab-authorship .files', { timeout: 90000 })
+      .should('be.visible');
+
+    cy.get('.line-number')
+      .first()
+      .should('have.css', 'color')
+      .then((firstColor) => {
+        cy.get('.line-number')
+          .each((el) => expect(el.css('color')).to.equal(firstColor));
+      });
```

Comment:
This test needs to be updated as it would pass even without the bug fix. The `line-number` CSS color here is still always grey (default line number colour) but the colour shown on the report is the CSS color of`hljs-comment` (dark grey) or `hljs-number` (blue). 

Please make sure the test is changed such that it fails before the fix, and passes after it. Thanks!

Reply:
My bad, I'll remember to do this for future tests!